app/js/questions_controller.js

```diff
@@ -43,6 +45,29 @@
     var tab_panels = container.querySelectorAll('[role="tabpanel"]');
     toggle_state(tab_panels, 'aria-hidden');
 
+    load_active_questions();
```

Comment:
We shouldn't load every time `set_active_tab` is called. This causes too much HTTP traffice and also messes the spinner state.